### PR TITLE
internal: deprecate 1.6-1.8

### PR DESCRIPTION
**Pull request by jadekler**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 sudo: false
 language: go
 go:
-  - 1.6
-  - 1.7
+  - 1.9
+  - 1.10
```
Comment (pongad) [inline .travis.yml]: I think you need `"1.10"` with quotes. Without quotes, this is unfortunately parsed as 1.1

Comment (jadekler) [reply .travis.yml]: I've run into this exact review comment before, making me think it's some old-but-fixed behavior of travis? Either way, it seems ok to use it without strings (e.g. https://github.com/grpc/grpc-go/blob/master/.travis.yml#L13)

Comment (pongad) [reply .travis.yml]: The ".x" might have caused it to work. If you go to the CI page though, the CI reports that it tried to build against Go1.1 right?

Comment (jadekler) [reply .travis.yml]: Oh my gosh you're absolutely right. So sorry for not checking before! Fixed by applying the .x.

**Pull request merged**